apps/spruce/src/pages/spawn/spawnVolume/SpawnVolumeTable.tsx

```diff
@@ -113,9 +117,24 @@ const columns = [
     cell: ({ getValue, row }) => {
       const expiration = getValue();
       const { noExpiration } = row.original;
-      return noExpiration || !expiration
-        ? DoesNotExpire
-        : formatDistanceToNow(expiration);
+      const isUnexpirable = noExpiration || !expiration;
+      return (
+        <>
+          {isUnexpirable ? DoesNotExpire : formatDistanceToNow(expiration)}
+          {!isUnexpirable && row.original.hostID && (
+            <Tooltip
```

Comment:
I think you can use an [InfoSprinkle](https://www.mongodb.design/component/info-sprinkle/live-example) instead!